Dockerfile

```diff
@@ -3,11 +3,14 @@ FROM quay.io/ukhomeofficedigital/nodejs-base:v4.4.2
 RUN mkdir /public
 RUN yum clean all && \
   yum update -y -q && \
-  yum install -y -q git && \
   yum clean all && \
   rpm --rebuilddb && \
-  npm install -g npm@latest --loglevel warn
+  npm --loglevel warn install -g npm@3
 
+COPY package.json /app/package.json
+RUN npm cache clean
+RUN npm --loglevel warn install --production --no-optional
 COPY . /app
+RUN npm --loglevel warn run postinstall
```

Comment:
I just realised that @lennym thanks in this PR https://github.com/UKHomeOffice/right-to-rent-check/pull/13

I'm just trying to get travis to run the app and use the app to run the acceptance tests